appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -726,6 +752,22 @@ public interface OdeMessages extends Messages, AutogeneratedOdeMessages {
   @Description("Switch to disable OpenDyslexic")
   String disableOpenDyslexic();
 
+  @DefaultMessage("Switch To New Layout")
+  @Description("Switch to enable new layouts")
+  String enableNewLayout();
+
+  @DefaultMessage("Switch To Old Layout")
+  @Description("Switch to disable new layouts")
+  String disableNewLayout();
+
+  @DefaultMessage("Use Dark Theme")
+  @Description("Switch to enable dark theme")
+  String enableDarkTheme();
+
+  @DefaultMessage("Use Light Theme")
+  @Description("Switch to disable dark theme")
+  String disableDarkTheme();
```

Comment:
Note earlier comments about when the dark theme is being introduced.

Reply:
Dark theme appeared in the project explorer prototype, but it isn't included in this release. Jennet has this feature mostly complete for a future release, but we need to consult about color choices.

I think we should keep the internationalization here, though, because it's coming.